testing/upload_validation.ts

```diff
@@ -717,7 +717,8 @@ const syncTableSchema = z.union([genericDynamicSyncTableSchema, genericSyncTable
 const unrefinedPackVersionMetadataSchema = zodCompleteObject<PackVersionMetadata>({
   version: z
     .string()
-    .regex(/^\d+(\.\d+){0,2}$/, 'Pack versions must use semantic versioning, e.g. "1", "1.0" or "1.0.0".'),
+    .regex(/^\d+(\.\d+){0,2}$/, 'Pack versions must use semantic versioning, e.g. "1", "1.0" or "1.0.0".')
+    .optional(),
```

Comment:
Hmm, is this actually needed? We still want a version to be specified when a pack is actually uploaded.

Where did you get errors without this change? We should see what other options we can come up to work around those, so we don't sacrifice the requirement that when you finally upload a pack version to persist, it needs to have a version.

Reply:
This failed when running `coda validate manifest.ts`. I guess I thought that it would be confusing if we made the version autogenerated but error-ed out in the validation step.